docs/source/user-guide/guide_harvest.rst

```diff
@@ -81,11 +81,37 @@ Run the harvest
 
 View the status of the harvest
 ------------------------------
-Navigate to *admin/dkan/harvest* to view the status of the extraction,
+
+  .. prompt:: bash $
+
+    drush dkan:harvest:status myHarvestId
+
+You can also navigate to *admin/dkan/harvest* to view the status of the extraction,
 the date the harvest was run, and the number of datasets that were added
 by the harvest. By clicking on the harvest ID, you will also see specific
 information about each dataset, and the status of the datastore import.
 
+Revert a harvest
+----------------
+
+  .. prompt:: bash $
+
+    drush dkan:harvest:revert myHarvestId
+
+This will delete all of the datasets listed in the specified harvest ID. Any referenced
+terms will be set to the orphaned state. And distributions from the harvest will be unpublished.
```

Comment:
Is this supposed to be "Any referenced terms will be set to the orphaned state and distributions from the harvest will be unpublished." or "Any referenced terms will be set to the orphaned state. Any distributions from the harvest will be unpublished."?

Reply:
@rovcase I think it works either way, if you think it reads better with 'Any' I can update it.

Reply:
Nope- works for me if it works for you, thanks!